source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -59,6 +59,23 @@ Insert a Document
 -----------------
 
 Use the ``InsertOne()`` function to insert a single document into a collection.
+You can modify the behavior of ``InsertOne()`` by constructing and passing
+an optional ``InsertOneOptions`` struct.
+
+The available options to tune with ``InsertOneOptions`` are:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Option
+     - Default
```

Comment:
Nit: Can default be the second line in the description column?  I thought it was visually better like how I did it for [the delete page](https://github.com/mongodb/docs-golang/pull/59/files#diff-2d78896fb8d9c69d30436f33745e05aa3d3fe135b5cb9f6141c980d5aeb69859R69-R70) since it gives a bit more horizontal space.